README.md

```diff
@@ -1,2 +1,28 @@
 # mentor-matcher
 A website for mentees to find mentors and mentors to offer their wisdom to others
+
+## Local Installation
+
+This project uses GNU Make to automate the setup process. Ensure that you have a `.env` file in the 
+root directory of the project. See `.env.example` for an example of the required environment 
+variables.
```

Comment:
I guess we'd also need to set up a new auth0 project per dev and populate the env file with values from the auth0 application settings page?

Reply:
Will be covered in issue #1